docker_explorer/lib/aufs.py

```diff
@@ -20,7 +20,7 @@
 import json
 import os
 
-from lib.storage import Storage
+from docker_explorer.lib.storage import Storage
```

Comment:
not sure which style you want to stick to (maybe something for a follow up CL) but I would recommend importing at module level not at class level

```
from docker_explorer.lib import storage
```

Reply:
Done